hedera-node/test-clients/src/main/java/com/hedera/services/bdd/spec/infrastructure/providers/ops/hollow/RandomAccountUpdateHollow.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright (C) 2024 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hedera.services.bdd.spec.infrastructure.providers.ops.hollow;
+
+import static com.hedera.services.bdd.spec.transactions.TxnVerbs.cryptoUpdate;
+import static com.hederahashgraph.api.proto.java.ResponseCodeEnum.INVALID_SIGNATURE;
+
+import com.hedera.services.bdd.spec.HapiSpecOperation;
+import com.hedera.services.bdd.spec.infrastructure.EntityNameProvider;
+import com.hedera.services.bdd.spec.infrastructure.providers.ops.crypto.RandomAccountUpdate;
+import com.hedera.services.bdd.spec.transactions.crypto.HapiCryptoUpdate;
+import com.hederahashgraph.api.proto.java.AccountID;
+import com.hederahashgraph.api.proto.java.Key;
+import java.util.Optional;
+
+public class RandomAccountUpdateHollow extends RandomAccountUpdate {
```

Comment:
```suggestion
public class RandomAccountUpdateWithSigner extends RandomAccountUpdate {
```

Reply:
Renamed to RandomAccountUpdateHollowAccount -> since currently the method expects only Invalid signature and passing a signer that is not a hollow account we make us also pass outcomes in constructor which which will not make the code better imo.